8bitdo-firmware.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+# based on https://ladis.cloud/blog/posts/firmware-update-8bitdo.html
+
+import requests,json,sys,os
+import urllib.request
+
+baseurl = "http://dl.8bitdo.com:8080"
+
+things = {}
```

Comment:
`firmwares`?

Reply:
Probably more like `products` (because each item is one thing 8BitDo sells, with a sublist of firmwares).